### PR TITLE
feat: upgrade to soroban-sdk v11.0.1 and update contract interfaces

### PR DESCRIPTION
**Pull request by mootz12**

### CODE REVIEW

File: src/backstop/backstop_client.ts [hunk 1]
```diff
@@ -183,7 +184,7 @@ export class BackstopClient {
     );
   }
 
-  async updateEmissionCycle(
+  async gulpEmissions(
```
Comment (Ryang-21) [inline src/backstop/backstop_client.ts:187]: Should the sdk also contain gulp pool emissions?

File: src/backstop/backstop_config.ts [hunk 2]
```diff
@@ -46,6 +48,7 @@ export class BackstopConfig {
     const backstopConfigEntries =
       (await rpc.getLedgerEntries(contractInstanceDataKey, lpValueDataKey, rewardZoneDataKey))
         .entries ?? [];
+    console.log(backstopConfigEntries);
```
Comment (Ryang-21) [inline src/backstop/backstop_config.ts:51]: fragment

File: src/emitter/emitter_client.ts [hunk 3]
```diff
@@ -76,26 +87,44 @@ export class EmitterClient {
...
+    network: Network,
+    txOptions: TxOptions
   ): Promise<ContractResult<undefined>> {
     return await invokeOperation<undefined>(
       source,
       sign,
       network,
       txOptions,
       () => undefined,
-      this.contract.call('drop', ...this.spec.funcArgsToScVals('drop', { new_backstop_id }))
```
Comment (Ryang-21) [inline src/emitter/emitter_client.ts:98]: Was the drop function intentionally removed?

**Review by Ryang-21 — CHANGES_REQUESTED** [review]
Had a few questions that I left as comments

File: src/backstop/backstop_client.ts (hunk 1 above)
Comment (mootz12) [reply src/backstop/backstop_client.ts:187]: No, this can't be called by anyone but the pool

File: src/emitter/emitter_client.ts (hunk 3 above)
Comment (mootz12) [reply src/emitter/emitter_client.ts:98]: Yes, this function is not on the emitter anymore/

**Review by Ryang-21 — APPROVED** [review]
lgtm

**Pull request merged**